components/ww3/cime_config/buildnml

```diff
@@ -50,8 +51,9 @@ def buildnml(case, caseroot, compname):
     # Verify wav grid is supported
     #--------------------------------------------------------------------
 
-    wav_grid_supported = ("null", "wQU225EC60to30sp32x36", "wQU225EC60to30", "wQU100", "wQU225EC30to60E2r2")
-    expect(wav_grid in wav_grid_supported, "WAV_GRID '{}' is not supported in ww3. Choose from: '{}'".format(wav_grid,' ,'.join(wav_grid_supported)))
+    wav_grid_supported = ("null", "wQU225EC60to30sp32x36", "wQU225EC60to30sp50x36", "wQU100sp50x36", "wQU225EC30to60E2r2sp50x36","wQU225EC30to60E2r2sp36x36","wQU225EC30to60E2r2sp25x36")
+    # expect((wav_grid.join(wav_spec)) in wav_grid_supported, "Combination of WAV_GRID {} and WAV_SPEC {} is not supported in ww3. Choose from: '{}'".format(wav_grid,wav_spec,wav_grid_supported) )
+      # this is throwing an error - comment out for now.
```

Comment:
I think the issue here is that `join` in python works a little differently than this. You can either do this as: `wave_grid+wave_spec` or `''.join([wav_grid,wav_spec])`

Reply:
thanks! I'll fix this